hammer_cli_foreman_tasks.gemspec

```diff
@@ -20,5 +20,5 @@ DESC
   s.files = Dir["{lib,config}/**/*", "LICENSE", "README.md"]
 
   s.add_dependency "powerbar", "~> 1.0.11"
-  s.add_dependency "hammer_cli_foreman", "> 0.1.1", "< 0.4.0"
+  s.add_dependency "hammer_cli_foreman", "~> 0.4"
```

Comment:
This increases the minimum to 0.4, which is probably not necessary as there hasn't been a code change to warrant it.  I'd suggest just keeping the old style but changing the second part to `< 1.0.0`.